api/app_analytics/tasks.py

```diff
@@ -35,6 +35,32 @@ def populate_bucket(
         populate_feature_evaluation_bucket(bucket_size, run_every, source_bucket_size)
 
 
+@register_recurring_task(
+    run_every=timedelta(days=1),
+)
+def clean_up_old_analytics_data():
+    # delete raw analytics data older than `RAW_ANALYTICS_DATA_RETENTION_DAYS`
+    APIUsageRaw.objects.filter(
+        created_at__lt=timezone.now()
+        - timedelta(days=settings.RAW_ANALYTICS_DATA_RETENTION_DAYS)
+    ).delete()
```

Comment:
Are we worried about model record counts for this task?

Reply:
No, those tables are not that huge